code/modules/ghostroles/instantiator.dm

```diff
@@ -130,6 +134,11 @@
 	// 		new_name = random_unique_name()
 	H.fully_replace_character_name(H.real_name, new_name)
 
+/**
+ * called after the mob is instantiated and the player is transferred in
+ */
+/datum/ghostrole_instantiator/proc/AfterSpawn(mob/created, list/params)
```

Comment:
this entire proc needs to go near the other proc stubs on base /datum/ghostrole instantiator instead of being so close to the latter part of the file